Gems/EMotionFX/Code/CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ ly_add_target(
     COMPILE_DEFINITIONS
         PUBLIC
             EMFX_DEVELOPMENT_BUILD
+            EMFX_ANIMGRAPH_PROFILER_ENABLED
```

Comment:
I thought about that as well. But I personally think the EMFX_DEVELOPMENT_BUILD define is too general. For example, user could want to still use the analysisor code to observe the performance of a non-development build.

Reply:
What would be your suggestion for the best way to fix this @lumberyard-employee-dm? We'd essentially like to have this on/enabled by default in Debug and Profile configurations but disabled in Release. It would be possible for users to set it on in Release if they really wanted it, but it shouldn't be on by default. I believe @lawsonamzn suggested a generator expression might be the best way to handle this? Something like...

```
$<$<NOT:$<CONFIG:release>>:EMFX_ANIMGRAPH_PROFILER_ENABLED>
```